Weather/Models/DailyWeatherData.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DailyWeatherData.swift
+//  Weather
+//
+//  Created by Alex on 31.07.22.
+//
+
+import Foundation
+
+struct DailyWeatherData: Codable, Identifiable {
+    var dt: Int
+    var temp: TemperatureData
+    var weather: [WeatherDetail]
+    
+    enum CodingKey: String {
+        case dt
+        case temp
+        case weather
+    }
+    
+    init() {
+        dt = 0
+        temp = TemperatureData(min: 0.0, max: 0.0)
+        weather = [WeatherDetail(main: "", description: "", icon: "")]
+    }
```

Comment:
Why do you need an empty data object? It's better to use an optional and set `nil` in this case

Reply:
Because i want be able to change default data if I needed